pkgs/development/python-modules/sphinxcontrib-openapi/default.nix

```diff
@@ -1,26 +1,46 @@
 { lib
 , buildPythonPackage
-, fetchPypi
+, deepmerge
+, fetchFromGitHub
+, fetchpatch
 , isPy27
 , setuptools-scm
-, m2r
-, pyyaml
 , jsonschema
+, picobox
+, pyyaml
+, sphinx-mdinclude
 , sphinxcontrib_httpdomain
 }:
 
 buildPythonPackage rec {
   pname = "sphinxcontrib-openapi";
-  version = "0.7.0";
+  version = "unstable-2022-08-26";
   disabled = isPy27;
```

Comment:
This shoudl be pythonOlder "3.6"

Reply:
python 3.6 doesn't exist either. It works for all remaining non-python 2.7 versions in nixpkgs. Which should be fine.